daemon/cluster/noderunner.go

```diff
@@ -163,6 +165,46 @@ func (n *nodeRunner) handleControlSocketChange(ctx context.Context, node *swarmn
 	}
 }
 
+func (n *nodeRunner) watchClusterEvents(ctx context.Context, conn *grpc.ClientConn) {
+	if n.cluster == nil {
+		return
+	}
+	client := swarmapi.NewStoreClient(conn)
+	watch, err := client.Watch(ctx, &swarmapi.WatchRequest{
+		Entries: []*swarmapi.WatchRequest_WatchEntry{
+			{
+				Kind:   "node",
```

Comment:
Would it be useful for `Watch` to support something like `Kind: "*"`?

In general I want people to filter for the specific events of interest, but the Docker events API is a bit of an unusual case.